SPEC.md

```diff
@@ -490,6 +491,10 @@ The App Container specification defines an HTTP-based metadata service for provi
 The ACE must provide a Metadata service on the address given to the applications via the `AC_METADATA_URL` [environment variable](#execution-environment).
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
+Clients must also authenticate by use of a bearer token.
```

Comment:
@eyakubovich I like to reference existing standards as much as possible when calling out things like "a bearer token"; AIUI that concept is not exactly genericised yet beyond OAuth...